docs/docs/tailwind-css.md

```diff
@@ -20,9 +20,11 @@ You have to install and configure Tailwind for both of these methods, so this gu
 1. Install Tailwind
 
 ```shell
-npm install tailwindcss --save-dev
+npm install tailwindcss@0.7.4 --save-dev
 ```
 
+> Note: This version of this guide doesn't work with Tailwind 1.0 and above. The Gatsby Team is working on this.
+
 2. Generate Tailwind config file
```

Comment:
Tailwind doesn't need a config file anymore as it ships with a default now - so we can remove this step

Reply:
I would maybe make this an optional step, as its optional in the Tailwind docs, but not remove it entirely. The tailwind config is still useful for customizing tailwind and for reducing bundle sizes if users don't want to/can't use PurgeCss.

But I'd also replace `'./node_modules/.bin/tailwind init'` with `npx tailwind init` to match the instructions in the docs. The current instructions throw an "unsigned powershell script" warning and requires users to change their powershell security settings to run it (at least with tailwind@1.04). I was going to open a PR to change the command to npx when I saw this one, and didn't want to open a second PR on the same doc.